multiclusterhub/charts/volsync-addon-controller/templates/volsync-addon-controller-serviceaccount.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ template "volsync-addon-controller.fullname" . }}
+  labels:
+    app: {{ template "volsync-addon-controller.name" . }}
+    chart: {{ template "volsync-addon-controller.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/name: {{ template "volsync-addon-controller.name" . }}
+    helm.sh/chart: {{ template "volsync-addon-controller.chart" . }}
+    component: volsync-addon-controller-serviceaccount
```

Comment:
Missing newline at end of last line.

https://stackoverflow.com/a/729795/1437822